modules/core/core-git/src/main/java/org/eclipse/dirigible/core/git/utils/RemoteUrl.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2022 SAP SE or an SAP affiliate company and Eclipse Dirigible contributors
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-FileCopyrightText: 2022 SAP SE or an SAP affiliate company and Eclipse Dirigible contributors
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.dirigible.core.git.utils;
+
+import org.eclipse.jgit.api.RemoteSetUrlCommand;
+import org.eclipse.jgit.lib.Repository;
+
+public class RemoteUrl extends RemoteSetUrlCommand {
+    protected RemoteUrl(Repository repo) {
+        super(repo);
+    }
+    public static RemoteUrl getRemoteUrl(Repository repo) {
+        return new RemoteUrl(repo);
+    }
+}
```

Comment:
I don't see the benefit of having this class, this seems like over engineering, as there is no added functionality to this abstraction. Can't we use `new RemoteSetUrlCommand(repo)` where needed?

Reply:
RemoteSetUrlCommand jgit class has a protected constructor that's why we need RemoteUrl class.
RemoteUrl class itself is approved as proposed by @ThuF